### ISSUE TITLE
`display` not forcing a view update when "view updates" is "continuous"

### ISSUE THREAD

**Issue by alan-isaac** [NetLogo/NetLogo#1729]
The documentation for `display` says:
"Causes the view to be updated immediately. (Exception: if the user is using the speed slider to fast-forward the model, then the update may be skipped.)"

I interpret this to mean that if the speed slider is left at normal, then `display` will always force an update, even when "View updates" is left at the "continuous" default.  I am pretty sure this is not happening: if the model runs very fast, the `display` requests are skipped.  On the other hand, if "view updates" is changed to "on ticks", then `display` appears to be honored.  If I am correct, is this a documentation bug or an implementation bug?  (Or am I misunderstanding the documentation?)

**Comment by SethTisue** [comment]
I haven't investigated, but implementation bug seems highly plausible. we went all-in on tick-based updates years ago, continuous updates are quite neglected (and aren't supported at all in NetLogo Web)

**Comment by ERSUCC** [comment]
Maybe I'm misunderstanding the issue, but I don't see an issue with `display` being skipped if the model is already running fast. If you're getting a high frame rate, wouldn't the `display` call be mostly irrelevant?

**Comment by alan-isaac** [comment]
The [documentation of `display`](https://ccl.northwestern.edu/netlogo/docs/dictionary.html#display) explcitly states that it *forces* a view update, and this is very important functionality is some models.

**Comment by ERSUCC** [comment]
Yes, but if the model is running fast enough, a view update may already be scheduled for the frame where `display` is called, making it a redundant call. This is especially the case for continuous updates. Maybe I'm misunderstanding how fast you mean by "the model runs very fast". Could you clarify exactly what you mean?

**Comment by alan-isaac** [comment]
It can be desirable to force a View update at specific model states, and the documentation of `display` indicates that this should be possible, even with continuous updates: "ordinarily NetLogo is free to skip some view updates, so that fewer total updates take place, so that models run faster. This command lets you force a view update".

**Comment by SethTisue** [comment]
@alan-isaac I think in order to argue for this, to be effective you probably have to also argue for what your use case for continuous updates even is, since continuous updates are a poorly supported legacy feature from another, very early era of NetLogo's design
